tensorboard/webapp/metrics/views/card_renderer/scalar_card_test.ts

```diff
@@ -2645,7 +2640,7 @@ describe('scalar card', () => {
           },
         },
       });
-      store.overrideSelector(getMetricsCardRangeSelectionEnabled, false);
+      store.overrideSelector(getMetricsRangeSelectionEnabled, false);
```

Comment:
Because `linkedTime` is enabled throughout this test suite the global range selection value should be used